src/Compilers/CSharp/Portable/DocumentationComments/DocumentationCommentIDVisitor.PartVisitor.cs

```diff
@@ -147,7 +147,12 @@ public override object VisitTypeParameter(TypeParameterSymbol symbol, StringBuil
 
                 // Is this a type parameter on a type?
                 Symbol containingSymbol = symbol.ContainingSymbol;
-                if (containingSymbol.Kind == SymbolKind.Method)
+                if (containingSymbol is null)
+                {
+                    // For instance a type parameter in cref (CrefTypeParameterSymbol)
+                    builder.Append('`');
```

Comment:
>builder.Append('`') [](start = 20, length = 19)

If this is a type parameter used in a reference to a nested type, we probably should do something about ```ordinalOffset```.